### DISCUSSION TITLE
Error on make command

### DISCUSSION THREAD

**Discussion by nonfungibletunji** [toucansites/toucan#14] [Q&A]
I am have the sample files installed in directory, i am using the command line only, not xcode

``` toucan generate ./src ./docs --base-url "http://localhost:3000/" ``` causes  the following error

Error: The file “index.md” couldn’t be opened because there is no such file.

**Comment by tib** [comment]
I will invetigate this. Were you using the samples from the blog post? 🤔

**Comment by nonfungibletunji** [comment]
Zip from the toucan sites repo.Where should I run the generate command in the content folder directory , thats the only place that has a index.md file.sight media lab.On Oct 1, 2024, at 12:19 PM, Tibor Bödecs ***@***.***> wrote:﻿
I will invetigate this. Were you using the samples from the blog post? 🤔

**Reply by nonfungibletunji** [reply]
still not working, 

which toucan 
/usr/local/bin/toucan 


the index.md still not found

Error: The file “index.md” couldn’t be opened because there is no such file.
